docs/src/main/sphinx/installation/cli.rst

```diff
@@ -287,3 +292,129 @@ To view debug information, including the stack trace for failures, use the
     at java.base/java.lang.Thread.run(Thread.java:834)
     select count(*) from tpch.tiny.nations
 
+.. _cli-options:
+
+Options
+-------
+
+The following table lists all supported command line options for the Trino CLI
+in alphabetical order:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--access-token``
+    - Access token for :ref:`cli-jwt-auth`.
+  * - ``--catalog``
+    - Default catalog.
+  * - ``--client-info``
+    - Extra information about client making query.
+  * - ``--client-request-timeout``
+    - Client request timeout (default: 2m).
+  * - ``--client-tags``
+    - Client tags.
+  * - ``--debug``
+    - Enable debug information.
+  * - ``--disable-compression``
+    - Disable compression of query results.
+  * - ``--execute``
+    - Execute specified statements and exit.
+  * - ``--external-authentication``
+    - Enable external authentication for :ref:`cli-external-sso-auth`.
+  * - ``--external-authentication-redirect-handler``
+    - External authentication redirect handlers: DESKTOP_OPEN, SYSTEM_OPEN,
+      PRINT, OPEN, ALL (default: ALL).
+  * - ``--extra-credential``
+    - Extra credentials (property can be used multiple times; format is
+      key=value).
+  * - ``-f`` ``--file``
+    - Execute statements from file and exit.
+  * - ``-h`` ``--help``
+    - Show this help message and exit.
+  * - ``--http-proxy``
+    - HTTP proxy to use for server connections.
+  * - ``--ignore-errors``
+    - Continue processing in batch mode when an error occurs (default is to exit
+      immediately).
+  * - ``--insecure``
+    - Skip validation of HTTP server certificates (should only be used for
+      debugging).
+  * - ``--keystore-path``
+    - The location of the Java Keystore file that is used to secure TLS. Connect
+      to a cluster that uses :ref:`cli-certificate-auth`.
+  * - ``--keystore-password``
+    - The password for the keystore. This must match the password you specified
+      when creating the keystore. Connect to a cluster that uses
+      :ref:`cli-certificate-auth`.
+  * - ``--keystore-type``
+    - Keystore type
+  * - ``--krb5-config-path``
+    - Kerberos configuration files.
```

Comment:
All these need to link to the kerberos auth section

Reply:
Just so you know, I deleted the links to that general section, initially because I was waiting until we add the content from the other Kerberos doc. I'm not sure if the links will go to specific content.